### ISSUE TITLE
Add accessibility language to Sinopia footer

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#3748]
Please add this language to the Sinopia footer. It should appear as the last sentence, but before the "Sinopia [version #]"

If you cannot access content or use features on this website due to a disability, please report your accessibility issue.

"report your accessibility issue" links to: mailto:library-accessibility-contact@lists.stanford.edu

Thank you!
